iot2000.coffee

```diff
@@ -0,0 +1,35 @@
+deviceTypesCommon = require '@resin.io/device-types/common'
+{ networkOptions, commonImg, instructions } = deviceTypesCommon
+
+module.exports =
+	version: 1
+	slug: 'IOT2000'
+	aliases: [ 'IOT2020', 'IOT2040' ]
+	name: 'Siemens IOT2000'
+	arch: 'i386'
+	state: 'preview'
+
+	instructions: commonImg.instructions
+
+	gettingStartedLink:
+		windows: 'http://docs.resin.io/#/pages/installing/gettingStarted-IOT2000.md#windows'
+		osx: 'http://docs.resin.io/#/pages/installing/gettingStarted-IOT2000.md#on-mac-and-linux'
+		linux: 'http://docs.resin.io/#/pages/installing/gettingStarted-IOT2000.md#on-mac-and-linux'
+
+	yocto:
+		machine: 'iot2000'
+		image: 'resin-image'
+		fstype: 'resinos-img'
+		version: 'yocto-morty'
```

Comment:
Any restriction for morty? Can we switch to pyro? meta-intel already supports pyro

Reply:
meta-iot2000 only supports morty

Reply:
Tried building it with pyro and had compile issues?

Reply:
I don't see any requirement in https://github.com/siemens/meta-iot2000 about build system version? Could you please detail, @telphan ?

Reply:
So did it compile with pyro? I know you were trying that a few days ago

Reply:
I am facing issues with that build. The pyro update will follow after this initial release.